scriptworker/constants.py

```diff
@@ -176,14 +176,16 @@
     # Map scopes to restricted-level
     'cot_restricted_scopes': frozendict({
         'firefox': frozendict({
-            'project:releng:balrog:release': 'all-release-branches',
-            'project:releng:beetmover:release': 'all-release-branches',
+            'project:releng:balrog:server:release': 'all-release-branches',
```

Comment:
Sounds good. Mea culpa for second statement, I copy-pasted and forgot to update.
👍 will follow-up with patch here.
Thanks!